File: beecrowd/1/1021/1021.py
dinheiroF = float(input())
dinheiro = 0
valores = [10000, 5000, 2000, 1000, 500, 200, 100, 50, 25, 10, 5, 1]
dinheiro = int(dinheiroF * 100)
print("NOTAS:")

for i in range(len(valores)):
    v = valores[i]
    valores[i] = dinheiro // v
    dinheiro -= valores[i] * v
    if i < 6:
        print("%d nota(s) de R$ %.2f" % (valores[i], v / 100))
    else:
        if i == 6:
            print("MOEDAS:")
        print("%d moeda(s) de R$ %.2f" % (valores[i], v / 100))

File: beecrowd/1/1021/1021.cs
using System;

class URI {

    static void Main(string[] args) {
        double dinheiroF;
        int dinheiro;
        int[] valores = {10000, 5000, 2000, 1000, 500, 200, 100, 50, 25, 10, 5, 1};
        dinheiroF = double.Parse(Console.ReadLine());
        dinheiro = (int) (dinheiroF * 100);
        Console.WriteLine("NOTAS:");
        for (int i = 0; i < 12; i++) {
            int v = valores[i];
            valores[i] = dinheiro / v;
            dinheiro -= valores[i] * v;
            if (i < 6) {
                Console.WriteLine("{0} nota(s) de R$ {1}", valores[i], ((double) v / 100).ToString("0.00"));
            } else {
                if (i == 6) {
                    Console.WriteLine("MOEDAS:");
                }
                Console.WriteLine("{0} moeda(s) de R$ {1}", valores[i], ((double) v / 100).ToString("0.00"));
            }
        }
    }
}

